qtop.py

```diff
@@ -1290,11 +1310,11 @@ def display_user_accounts_pool_mappings(self, wns_occupancy):
               colorize("  ('all' also includes those in C and W states, as reported by qstat)"
                             if options.CLASSIC else "  ('all' includes any jobs beyond R and W)", 'Gray_D')
 
-        print '   R +    Q /  all |       unix account [id] %(msg)s' % \
+        print '   R +    Q /  all | nodes |      unix account [id] %(msg)s' % \
```

Comment:
I am assuming you mean for the id column and not for the nodes column. Will try it.